smaller_constraints.py

```diff
@@ -0,0 +1,41 @@
+from ortools.sat.python import cp_model
+from ortools.sat.python.cp_model import IntVar
+import datetime
+
+from models.employees.employee import Employee
+from models.shifts.shift import Shift
+from models.shifts.shifts_types_enum import ShiftTypesEnum
+
+
+def add_at_most_one_employee_per_shift_constrain(constraint_model: cp_model.CpModel, worker_shifts_worked_on_a_day: list[IntVar]):
```

Comment:
why keep the old, non small constraint? remove that code. remove all reference to it in this PR. dont keep old legacy code in